SimPEG/electromagnetics/time_domain/simulation.py

```diff
@@ -23,9 +23,13 @@ class BaseTDEMSimulation(BaseTimeSimulation, BaseEMSimulation):
     Euler.
     """
 
-    def __init__(self, mesh, survey=None, dt_threshold=1e-8, **kwargs):
+    def __init__(
+        self, mesh, survey=None, dt_threshold=1e-8, store_factors=True, **kwargs
```

Comment:
Are we sure we want to set the default for `store_factors` to True? I'm pretty sure that people new to SimPEG won't fully consider the impact of this choice when simulating TDEM data.